x-pack/plugins/security_solution/public/detections/components/alerts_table/timeline_actions/use_add_to_case_actions.tsx

```diff
@@ -59,18 +60,31 @@ export const useAddToCaseActions = ({
       : [];
   }, [casesUi.helpers, ecsData, nonEcsData]);
 
-  const { activeStep, endTourStep, incrementStep, isTourShown } = useTourContext();
+  const { activeStep, incrementStep, isTourShown } = useTourContext();
 
   const afterCaseCreated = useCallback(async () => {
     if (isTourShown(SecurityStepId.alertsCases)) {
-      endTourStep(SecurityStepId.alertsCases);
+      // user could be on step 5 or 6 at this point, increment to 7 no matter what
+      incrementStep(SecurityStepId.alertsCases, AlertsCasesTourSteps.viewCase);
```

Comment:
Maybe we could create a new `setStep` for this one, using `incrementStep` may seem we are incrementing by 7, but we are not incrementing, we are setting a specific step value. If we change the function name we won't need the comment explaining what it does.